fastlane/lib/fastlane/helper/adb_helper.rb

```diff
@@ -38,7 +38,7 @@ def device_avalaible?(serial)
       def load_all_devices
         self.devices = []
 
-        command = [adb_path, "devices"].join(" ")
+        command = [adb_path.shellescape, "devices"].join(" ")
```

Comment:
how about doing the `.shellescape` only once in `initialize`?

Reply:
I decided against that because one could theoretically use the `adb_path` somewhere outside of the context of a command (e.g. for output, logging etc), and then you would not want it to be shellescaped. Afaik shellescaping is only useful if used in a command (on the shell...), so I thought we should not _always_ do that. Agree?

Reply:
good point.